src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -77,16 +94,9 @@ public static void CheckSparseColumnBit()
             // table name will be provided as an argument
             StringBuilder createBuilder = new StringBuilder("CREATE TABLE {0} ([ID] int PRIMARY KEY, [CSET] xml COLUMN_SET FOR ALL_SPARSE_COLUMNS NULL");
 
-            // TSQL to create the same table, but without the column set column and without sparse
-            // also, it has only 1024 columns, which is the server limit in this case
-            StringBuilder createNonSparseBuilder = new StringBuilder("CREATE TABLE {0} ([ID] int PRIMARY KEY");
```

Comment:
I wonder if this was for a reason, maybe worth enabling these cases instead of removing code?

Reply:
I can revert the clean up, enabling in this PR seems out of scope ?

Reply:
Yes that would be fine. Thanks!